misk-bom/build.gradle.kts

```diff
@@ -8,6 +8,11 @@ dependencies {
     // Jetty 10+ uses SLF4J 2.0-alpha, which is unstable. It works well with 1.7 substituted in.
     // See: https://github.com/eclipse/jetty.project/issues/5943
     api(Dependencies.slf4jApi)
+    project.rootProject.subprojects.forEach { subproject ->
+      if (subproject.name != "misk-bom") {
+        api(subproject)
+      }
+    }
```

Comment:
This just does what we do for OkHttp’s BOM
https://github.com/square/okhttp/blob/master/okhttp-bom/build.gradle.kts